monitoring/ingestion/ingestion-producer-alert.yaml

```diff
@@ -0,0 +1,74 @@
+x-inputs:
+  - name: namespace
+    type: constant
+  - name: job_ingestion_producer
+    type: constant
+  - name: replicas
+    type: constant
+
+groups:
+- name: IngestionProducer
+  rules:
+
+  - alert: IngestionProducerDown
+    Expr: sum(up{namespace="${namespace}", job="${job_ingestion_producer}"}) == 0
+    For:   "30s"
+    Labels:
+      severity: critical
+    Annotations:
+      zenko_service: backbeat-ingestion-producer
+      description: "Ingestion Producer pod has been down for 30 seconds"
+      summary: "Ingestion Producer service is down"
+
+  - alert: IngestionProducerDegraded
+    Expr: sum(up{namespace="${namespace}", job="${job_ingestion_producer}"}) < ${replicas}
+    For:   "30s"
+    Labels:
+      severity: warning
+    Annotations:
+      zenko_service: backbeat-ingestion-producer
+      description: "Ingestion Producer pod has been down for 30 seconds"
+      summary: "Ingestion Producer service is degraded"
+
+  - alert: IngestionProducerLagIncreasedBy3Percent
+    Expr: |
+      ((sum(metadata_journal_size-ingestion_read_offset+1) -
```

Comment:
Other job (service) might have the same metrics name. 
Please specify the `job_ingestion_producer` and `namespace`.